jbmc/regression/jbmc/nondet-array-size/test.desc

```diff
@@ -0,0 +1,16 @@
+CORE
+TestClass.class
+--function "TestClass.minimalJbmc" --unwind 5 --max-nondet-string-length 10 --java-unwind-enum-static --trace --json-ui
+^EXIT=10$
+^SIGNAL=0$
+VERIFICATION FAILED
+--
+--
+Verify that the trace produced when creating an array whose size is determined
```

Comment:
"Verify that when an array's size is determined by an array input, the resulting trace is valid"?